infinispan/src/test/resources/infinispan-local.xml

```diff
@@ -23,33 +22,29 @@
         <!-- *************************** -->
         <!--   Default cache settings    -->
         <!-- *************************** -->
-        <local-cache name="DEFAULT">
-            <transaction mode="NON_XA" />
+        <local-cache-configuration name="default-template">
+            <locking isolation="READ_COMMITTED" />
+            <transaction mode="NON_DURABLE_XA" transaction-manager-lookup="org.infinispan.transaction.lookup.JBossStandaloneJTAManagerLookup"/>
```

Comment:
Why did we change the transaction mode? I'd expect `NON_XA` to still be a reasonable default?

Reply:
Also wondering if `transaction-manager-lookup` is necessary?

Reply:
I tried it and neither seems to be necessary according to our tests. I'm not sure though :)